internal/backup.go

```diff
@@ -109,6 +142,16 @@ func (backup *Backup) UploadMetadata(metadataDto interface{}) error {
 
 func (backup *Backup) UploadSentinel(sentinelDto interface{}) error {
 	sentinelPath := backup.getStopSentinelPath()
+	if CommonAllowedSettings[UseSerializedJSONSetting] {
+		r, w := io.Pipe()
+		go func() {
+			err := einJSON.Marshal(sentinelDto, w)
```

Comment:
You pass the io.Writer to the `einJSON.Marshal` func and not closing it. It results in never ending read from the uploader. 

Change signature of  einJSON.Marshal  to  `func Marshal(v interface{}, buf io.WriteCloser) error` and close the `w` after `e.marshal` completes.

Reply:
Done in latest release